P5/Source/Specs/att.datable.xml

```diff
@@ -49,6 +58,20 @@ $Id$
           </sch:rule>
         </constraint>
       </constraintSpec>
+      <constraintSpec scheme="schematron" ident="calendar-deprecated">
+        <constraint>
+          <sch:rule context="tei:acquisition[@calendar] | tei:affiliation[@calendar] | tei:age[@calendar] | tei:altIdentifier[@calendar] | tei:application[@calendar] | tei:author[@calendar] | tei:binding[@calendar] | tei:birth[@calendar] | tei:bloc[@calendar] | tei:change[@calendar] | tei:climate[@calendar] | tei:conversion[@calendar] | tei:country[@calendar] | tei:creation[@calendar] | tei:custEvent[@calendar] | tei:death[@calendar] | tei:district[@calendar] | tei:editor[@calendar] | tei:education[@calendar] | tei:event[@calendar] | tei:faith[@calendar] | tei:floruit[@calendar] | tei:funder[@calendar] | tei:gender[@calendar] | tei:geogFeat[@calendar] | tei:geogName[@calendar] | tei:idno[@calendar] | tei:langKnowledge[@calendar] | tei:langKnown[@calendar] | tei:licence[@calendar] | tei:location[@calendar] | tei:meeting[@calendar] | tei:name[@calendar] | tei:nationality[@calendar] | tei:objectName[@calendar] | tei:occupation[@calendar] | tei:offset[@calendar] | tei:orgName[@calendar] | tei:origPlace[@calendar] | tei:origin[@calendar] | tei:persName[@calendar] | tei:persPronouns[@calendar] | tei:placeName[@calendar] | tei:population[@calendar] | tei:precision[@calendar] | tei:principal[@calendar] | tei:provenance[@calendar] | tei:region[@calendar] | tei:relation[@calendar] | tei:residence[@calendar] | tei:resp[@calendar] | tei:seal[@calendar] | tei:settlement[@calendar] | tei:sex[@calendar] | tei:socecStatus[@calendar] | tei:sponsor[@calendar] | tei:stamp[@calendar] | tei:state[@calendar] | tei:terrain[@calendar] | tei:title[@calendar] | tei:trait[@calendar] | tei:unitDecl[@calendar] | tei:unitDef[@calendar]">
+            <sch:let name="me_phrase"
+              value="concat( 'Attribute calendar on ', local-name(.))"/>
+            <sch:assert test="'2024-11-11' cast as xs:date ge current-date()">
+              <sch:value-of select="concat( $me_phrase, ' construct is outdated (as of 2024-11-11); ODD processors may ignore it, and its use is no longer supported' )"/>
```

Comment:
> Should we add a link to the GH issue so that users can see the discussion? #2045

I do not think it is a good idea for the _Guidelines_ to refer to specific issues or discussions except perhaps as a footnote.